ClinGen/clincoded
# Connects to #990. Connects to #837. Add MOI button to VCI

#### Connects to #837:

* Added 'Evidence View' header text to VCI when in Evidence view mode

![image](https://cloud.githubusercontent.com/assets/4326866/18693432/7d4be7f0-7f56-11e6-8d9b-44561728e52f.png)

#### Connects to #990:

* Added `modeInheritance` field to `interpretation` object
* Moved mode of inheritance list out of `create_gene_disease.js` to external json file `modes_of_inheritance.json`
* Added `Oligogenic inheritance (HP:0010983)` value to modes of inheritance list
* Re-wire `create_gene_disease.js` to use `modes_of_inheritance.json` to generate dropdown list
* Update schemas for `interpretation` and `gdm` objects to allow new Oligogenic value
* Added 'Inheritance (+)` button to VCI, which switches to `Inheritance (pencil)` when mode of inheritance has been associated w/ interpretation
* Modified `Associate Disease` button to `Disease (+)`/`Disease (pencil)` button
* Added associate mode of inheritance modal to VCI
* Made it so user can remove Disease and Inheritance associations to interpretations
* Updated title and title styles to display MOI values
* Added and updated history styles to support new actions on dashboard

![image](https://cloud.githubusercontent.com/assets/4326866/18693699/a097d21c-7f58-11e6-9196-41a8bf408c9f.png)

![image](https://cloud.githubusercontent.com/assets/4326866/18693684/899e2624-7f58-11e6-95e6-d5f3d15b6efb.png)

![image](https://cloud.githubusercontent.com/assets/4326866/18693696/9423b8d4-7f58-11e6-9cac-76f46b48964f.png)

![image](https://cloud.githubusercontent.com/assets/4326866/18693705/b20b1d10-7f58-11e6-9dd2-8c5b92c9d59f.png)

![image](https://cloud.githubusercontent.com/assets/4326866/18693715/cd427e66-7f58-11e6-9c9e-ac23db4a863c.png)


*Testing:*

1. Get to VCI
2. Verify that the page title says 'Evidence View'
3. Begin interpretation and confirm that the two new Disease/Inheritance buttons exist
4. Associate disease/inheritance, and confirm that the modals work as expected, and that the buttons and VCI subtitles update properly
5. Modify disease/inheritance, and confirm that the operations work as expected, and that the buttons and VCI subtitles update properly
6. Remove disease/inheritance, and confirm that the operations work as expected, and that the buttons and VCI subtitles update properly
7. Go to the Dashboard and confirm that history items for these operations show up properly

*Cross-check testing:*

1. Go to create GDM page, and confirm that the MOI dropdown is populated properly, and that the new Oligogenic Inheritance entry exists
2. Create GDM to check that the page works properly
